backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -199,19 +199,23 @@ def _error_message(i: int, _url: str) -> dict:
             errors.append(_error_message(index, url))
 
 
-def verify_collection_body(body: dict, errors: list, allow_none: bool = False) -> None:
-    result = email_regex.match(body.get("contact_email", ""))
-    if not result and not allow_none:
-        errors.append({"name": "contact_email", "reason": "Invalid format."})
-
-    if not body.get("description") and not allow_none:  # Check if description is None or 0 length
-        errors.append({"name": "description", "reason": "Cannot be blank."})
-
-    if not body.get("name") and not allow_none:  # Check if name is None or 0 length
-        errors.append({"name": "name", "reason": "Cannot be blank."})
-
-    if not body.get("contact_name") and not allow_none:  # Check if contact_name is None or 0 length
-        errors.append({"name": "contact_name", "reason": "Cannot be blank."})
+def verify_collection_body(body: dict, errors: list) -> None:
+    def _check_if_blank(key) -> bool:
```

Comment:
I thought the leading `_` was a convention to indicate the method should be treated as private. I don't think this applies to methods defined in function scope. 🤔